source/com.squareup.okhttp3/okhttp-jvm/Transforms/Metadata.xml

```diff
@@ -0,0 +1,9 @@
+<metadata>
+    <remove-node path="/api/package[starts-with(@name, 'okhttp3.internal')]" />
+    <remove-node path="/api/package/class/method[starts-with(@name, '-')]" />
```

Comment:
Is this removing methods that start with the `-` character? Are there really methods named that, which won't be removed for other reasons?

Remove this if it doesn't make sense, or explain why it's here.